### DISCUSSION TITLE
no CLI available through Flatpak

### DISCUSSION THREAD

**Discussion by mscurtescu** [flameshot-org/flameshot#3025] [Q&A] [answered]
I installed Flameshot on Pop!_OS 22.04 (which is based on Ubuntu 22.04) using the native app store (Pop!_Shop). I checked and it was installed through Flatpak, running version 12.1.0:

```
Flameshot v12.1.0 ()
Compiled with Qt 5.15.5
linux: 6.0.6-76060006-generic
org.kde.Platform: 5.15-21.08
```

There is no `flameshot` CLI command, and no `.config/flameshot` either.

Installing through Snap does provide the CLI.

This  is probably a Flatpak configuration issue or limitation, but worth noting in case someone else runs into the same problem.

**Comment by mmahmoudian** [comment, marked as answer]
Flatpaks are working differently when it comes to CLI. Read more here (also the quick fix):

https://flameshot.org/docs/installation/installation-linux/#flatpak

> but worth noting in case someone else runs into the same problem.

Yes, but users are expected to:
1. Read the documentation
2. Know their own software stack and how it work. This is a general behaviour of Flatpak and has nothing to do with Flameshot.

Thanks for reporting this. Hopefully this thread helps some other users.
